build.sbt

```diff
@@ -80,7 +80,7 @@ lazy val common: Seq[Setting[_]] =
       val openModules =
         if (isJdk17orHigher) Seq("--add-opens=java.base/java.util.concurrent=ALL-UNNAMED")
         else Nil
-      "-Xms1G" :: "-Xmx1G" :: "-XX:MaxDirectMemorySize=256M" :: akkaProperties ++ openModules
+      "-Xms1G" :: "-Xmx16G" :: "-XX:MaxDirectMemorySize=256M" :: akkaProperties ++ openModules
```

Comment:
What is the reason for this, what is taking that much memory? If it's about reducing risk of gc that cause flakiness in starvation test it should probably bump both `-Xms` and `-Xmx`, but I doubt 16G would be any better than for example `4G`.

Reply:
It's the test loop where it tries to simulate blocking on disk, on our modern macs it will hit OOM with smaller heaps, way before it manages to trigger slow enough IO. I'm sure it could be possible to come up with some other strategy, but this was the shortest path to make it work. On a slower disk/machine it will anyway not reach those amounts of heap, Xms is still 1G.

Reply:
alright, thanks